dashboard/app/views/shared/_user_header.html.haml

```diff
@@ -13,9 +13,17 @@
         - if current_user.teacher? && (current_user.teacher_prize || current_user.teacher_bonus_prize)
           = link_to t('nav.user.prizes'), my_prizes_url
         = link_to t('nav.user.logout'), destroy_user_session_url
+    :javascript
+      if (typeof dashboard !== 'undefined') {
```

Comment:
Is this the case sometimes / do we want to fail silently here when not available?